src/components/Footer/Footer.scss

```
@@ -0,0 +1,55 @@
+@font-face {
+  font-family: 'KoPubDotumMedium';
+  src: url('https://cdn.jsdelivr.net/gh/projectnoonnu/noonfonts_two@1.0/KoPubDotumMedium.woff')
+    format('woff');
+  font-weight: normal;
+  font-style: normal;
+}
+
+footer {
+  font-family: 'KoPubDotumMedium';
+  position: fixed;
+  height: 270px;
+  width: 100%;
+  bottom: 0;
+  background-color: #fb8928;
+  section {
+    display: flex;
+    justify-content: space-between;
+    align-self: center;
+    width: 45%;
+    height: 270px;
+    margin: auto;
+    margin-top: 30px;
+    // background-color: pink;
+    color: linen;
+    .callNotice {
+      font-size: 20px;
```

```suggestion
footer {
  font-family: 'KoPubDotumMedium';
  position: fixed;
  height: 270px;
  width: 100%;
  bottom: 0;
  background-color: #fb8928;
  
  section {
    display: flex;
    justify-content: space-between;
    align-self: center;
    width: 45%;
    height: 270px;
    margin: auto;
    margin-top: 30px;
    // background-color: pink;
    color: linen;
    
    .callNotice {
      font-size: 20px;
```
위와 같이 가독성을 위해 셀럭터 시작할 때는 한 줄 띄워주시기 바랍니다.